packages/client/tests/functional/fulltext-search/__snapshots__/tests.ts.snap

```diff
@@ -23,7 +23,7 @@ Invalid \`.findMany()\` invocation in
   XX   const result = prisma.user
 → XX     .findMany(
 Error occurred during query execution:
-ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(MysqlError { code: 1064, message: "rpc error: code = InvalidArgument desc = syntax error, unexpected '-' (errno 1064) (sqlstate 42000) (CallerID: userData1): Sql: \\"select \`User\`.id, \`User\`.email, \`User\`.\`name\` from \`User\` where match(\`User\`.\`name\`) against (:vtg1 /* VARCHAR */ in boolean mode)\\", BindVars: {#maxLimit: \\"type:INT64 value:\\\\\\"10001\\\\\\"\\"vtg1: \\"type:VARCHAR value:\\\\\\"John <--> Smith\\\\\\"\\"}", state: "42000" })), transient: false })
+ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(MysqlError { code: 1064, message: "target: test-0000-00000000.0.primary: vttablet: rpc error: code = InvalidArgument desc = syntax error, unexpected '-' (errno 1064) (sqlstate 42000) (CallerID: userData1): Sql: \\"select \`User\`.id, \`User\`.email, \`User\`.\`name\` from \`User\` where match(\`User\`.\`name\`) against (:vtg1 /* VARCHAR */ in boolean mode)\\", BindVars: {#maxLimit: \\"type:INT64 value:\\\\\\"10001\\\\\\"\\"vtg1: \\"type:VARCHAR value:\\\\\\"John <--> Smith\\\\\\"\\"}", state: "42000" })), transient: false })
```

Comment:
@millsp I don't particularly like the extra `target: test-0000-00000000.0.primary: vttablet:` we get now, but I'm not opposed to merging this.